src/api/app/models/concerns/staging_project.rb

```diff
@@ -15,7 +15,7 @@ module StagingProject
   end
 
   HISTORY_EVENT_TYPES = [:staging_project_created, :staged_request, :unstaged_request].freeze
-  FORCEABLE_STATES = [:building, :failed, :testing].freeze
+  FORCEABLE_STATES = [:building, :failed, :testing, :acceptable].freeze
```

Comment:
I wonder why this isn't simply for all states?

Reply:
Because there are states you don't want to allow to force. Like `:empty`, `:review` or intermediate states like `:accepting`